### ISSUE TITLE
Add ability to run any source file through a command line option

### ISSUE THREAD

**Issue by Ok23** [anael-seghezzi/CToy#25]

**Comment by anael-seghezzi** [comment]
Yes why not, it should be easy to add. Let's think about the syntax.

**Comment by Ok23** [comment]
I do not think that there is a lot of space for a variety. If there is no second path argument, look for source in CToy application path src/main.c

**Comment by Ok23** [comment]
Maybe add optional arguments such as --inc for includes, --lib for static libs

**Comment by anael-seghezzi** [comment]
So, after thinking about it, using the command line to run any source is not an option because:
1: It will break compatibility of previous projects already using "ctoy_argc" and "ctoy_argv".
2: It will break the ability to compile with other compiler (gcc etc) with "tool/build_native" and CMake.
Because the CMake script will have no way to know which c files to compile.

The only option I see would be to add a "CMakeLists.txt" file to "src/" and parse it.
The default CMakeLists.txt would be `FILE(GLOB SOURCES main.c)` and could be modified.

If parsing a basic `FILE(GLOB SOURCES` is not too hard, handling the complete syntax (recursion, *, etc) is not as simple.
